msal4j-sdk/pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.microsoft.azure</groupId>
 	<artifactId>msal4j</artifactId>
-	<version>1.13.3</version>
+	<version>1.0.0-beta.1</version>
```

Comment:
This version is already released.
why is it marked to an already released version ?

Reply:
Talked offline about release versioning. Resolving it here.